app/views/events/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>All Events</h1>
+
+<br>
+
+<%= render "users/showtb" %>
```

Comment:
This partial should be put in events, after all, it contains past and upcoming events. `events/showtb`. You could also improve the naming, `showtb`.